tests/config_test.go

```diff
@@ -495,17 +500,20 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				Expect(err).ToNot(HaveOccurred())
 				defer expecter.Close()
 
-				res, err := expecter.ExpectBatch([]expect.Batcher{
+				res, err := tests.ExpectBatchWithValidatedSend(expecter, []expect.Batcher{
 					// mount iso Secret image
 					&expect.BSnd{S: "sudo su -\n"},
-					&expect.BExp{R: "\\#"},
```

Comment:
I wanted to write that if you want to drop the escape character you should use backticks `#`.
But then I realized I don't understand why "#" has to be escaped. Does "#" have any special meaning in go regex? @jean-edouard

Reply:
`#` is a weird one...
It doesn't seem to be documented as a special character (https://golang.org/pkg/regexp/syntax/), yet I'm almost certain I've seen it match the empty string!
https://regoio.herokuapp.com/ doesn't agree though...
I could be worth retesting that. (also, escaping it doesn't hurt)

Reply:
> I could be worth retesting that. (also, escaping it doesn't hurt)

I dropped it because I thought it could hurt:)  aren't we matching here against "\\#" since we are using double backslashes? 
with a single backslash it won't compile saying : 
`unknown escape sequence`

Reply:
Well that's different. There is a lot of confusion when using interpreted strings in conjonction with regexps. The problem is that they both support escape sequences, and both use the same escape character!
That's why strings used for regexps should be raw strings as much as possible, to lift one of the 2 interpretations.
In this case, regexp supports escaping # but not go's interpreted strings. The error you got is actually pretty explicit. When I said # was safe to escape, I meant safe to escape in regexps, not in go's interpreted strings!
If all that makes no sense, it only proves my point that we should switch to raw strings everywhere :)

Reply:
Thanks @jean-edouard  for the explanation.
will try to adopt the raw strings. `\\` confuses me.  I still Think that as general approach, to avoid confusion we should escape
only special characters as it could lead to mistakes, and I don't think # is a special char in this context. Is that OK with you?